src/index.ejs

```diff
@@ -46,49 +43,49 @@
       if (!useModal){
         document.getElementById("onfido-button").style.visibility = "hidden";
       }
-
       window.onload = function() {
-        var url = 'https://sdk-jwt-factory-staging.herokuapp.com/api'
+        window.onfidoOut = Onfido.init({
+          useModal: useModal,
+          buttonId: 'onfido-button',
+          onReady: function() {
+            /*callback for when */ console.log("successfully authorised")
+          },
+          onDocumentCapture: function(data) {
+            /*callback for when the*/ console.log("document has been captured successfully", data)
+          },
+          onFaceCapture: function(data) {
+            /*callback for when the*/ console.log("face capture was successful", data)
+          },
+          onComplete: function(data) {
+            /*callback for when */ console.log("everything is complete", data)
+            console.log(Onfido.getCaptures())
+          },
+          steps: [
+            {
+              type:'welcome',
+              options:{
+                title:'Open your new bank account'
+              }
+            },
+            {
+              type:'document',
+              options:{
+                useWebcam: options.useWebcam === "true"
+              }
+            },
+            'face',
+            'complete'
+          ]
+        })
+        var url = '<%= htmlWebpackPlugin.options.JWT_FACTORY %>'
```

Comment:
I think using the Define plugin in html files stopped working in webpack 2.0.

This workaround comes from:
https://github.com/webpack/webpack/issues/3105
https://stackoverflow.com/questions/39902197/how-can-i-pass-webpack-environment-variables-in-html